go.mod

```diff
@@ -4,8 +4,8 @@ go 1.24.0
 
 require (
 	github.com/chzyer/readline v1.5.1
-	github.com/consensys/gnark v0.12.0
-	github.com/consensys/gnark-crypto v0.17.0
+	github.com/consensys/gnark v0.14.0
+	github.com/consensys/gnark-crypto v0.19.0
```

Comment:
Same update should be applied to ZKP examples.

Reply:
Fixed.